server/store/sqlstore/store.go

```diff
@@ -886,17 +886,54 @@ func (s *SQLStore) GetStats() (*storemodels.Stats, error) {
 		return nil, err
 	}
 
-	query = s.getQueryBuilder().Select("count(id)").From("users").Where(sq.NotEq{"RemoteId": ""}).Where(sq.Like{"Username": "msteams_%"}).Where(sq.Eq{"DeleteAt": 0})
+	query = s.getQueryBuilder().Select("count(id)").From("users").Where(sq.And{
+		sq.Eq{"RemoteId": remoteID},
+		sq.Or{sq.Eq{"DeleteAt": 0}, sq.Eq{"DeleteAt": nil}},
+	})
 	row = query.QueryRow()
 	var syntheticUsers int64
 	if err := row.Scan(&syntheticUsers); err != nil {
 		return nil, err
 	}
 
+	var msTeamPrimary int64
+	var mmPrimary int64
+	query = s.getQueryBuilder().Select("preferences.value", "count(*)").
+		From("preferences").
+		LeftJoin(fmt.Sprintf("%s ON preferences.userid = %s.mmuserid", usersTableName, usersTableName)).
```

Comment:
nit: we can alias the `usersTableName` to avoid interpolations later:

```suggestion
		LeftJoin(fmt.Sprintf("%s u ON preferences.userid = u.mmuserid", usersTableName)).
```